js/circle.js

```diff
@@ -0,0 +1,13 @@
+class Circle extends Shape {
+    constructor(x, y, size, stepsX, stepsY) {
+        super(x, y, size, stepsX, stepsY);
+        this.stepsX = 0;
+        this.stepsY = random(-7, 7);
+    }
+
+    draw() {
```

Comment:
One cleaner way to do this is to have the draw function in the parent and call it from here extending it to draw whatever you want.
So in the parent it would be 
draw() {
 this.fillColor();
        this.move();
        }
        
        and in the children it would be 
        draw() {
        super.draw();
        circle(this.x, this.y, this.size);
        }